test/superlogger_test.rb

```diff
@@ -33,14 +33,13 @@ def output
     assert_not_nil Superlogger::VERSION
   end
 
-  test 'log format' do
+  test 'log format when session is not loaded' do
```

Comment:
Should there be a corresponding test for `log format when session is loaded`?

Reply:
Oops, I just saw https://github.com/moexmen/superlogger/pull/23:
> See the test cases, in particular the one titled log format when session is loaded.

Reply:
That test case cannot be added until the fix in #23: https://github.com/moexmen/superlogger/pull/23/commits/213002ab37bc53077ccc493c260256428aa0f31d